src/aiida_quantumespresso/workflows/pw/base.py

```diff
@@ -29,10 +29,11 @@ class PwBaseWorkChain(ProtocolMixin, BaseRestartWorkChain):
         'qe': qe_defaults,
         'delta_threshold_degauss': 30,
         'delta_factor_degauss': 0.1,
-        'delta_factor_mixing_beta': 0.8,
+        'delta_factor_mixing_beta': 0.5,
```

Comment:
So if we start from `0.4` then the next will be: 0.2, 0.1, 0.05, ... Nevertheless, `0.05` is rather uncommon. Shall we set something to reproduce e.g. `0.3, 0.2, 0.1, 0.05`? Or probably just set a list of them? Ok, here I don't consider starting from a different mixing beta. What do you thik?

Reply:
Actually coming back here - probably it is just best to leave `0.5` as a factor, son one explores quickly the possible betas. No need to stay too close I guess.